assets/src/edit-story/components/inspector/document/test/pageAdvancement.js

```diff
@@ -60,4 +62,50 @@ describe('PageAdvancementPanel', () => {
       },
     });
   });
+
+  it('should set Page Duration', async () => {
+    const { getByRole, updateStory } = setupPanel({
+      autoAdvance: true,
+    });
+    const element = getByRole('button', { name: 'Page Advancement' });
+    expect(element).toBeDefined();
+
+    const slider = getByRole('slider', { name: 'Default Page Duration' });
+
+    fireEvent.change(slider, {
+      target: { value: 0 },
+    });
+
+    await waitFor(() =>
+      expect(updateStory).toHaveBeenCalledWith({
+        properties: {
+          defaultPageDuration: 1,
+        },
+      })
+    );
+
+    fireEvent.change(slider, {
+      target: { value: 1 },
+    });
+
+    await waitFor(() =>
+      expect(updateStory).toHaveBeenCalledWith({
+        properties: {
+          defaultPageDuration: 1,
+        },
+      })
+    );
```

Comment:
This will always be true, because it was true before. The call queue does not reset unless you ask it to. If you want to validate, that exactly 1 new invocation has been made at it had the value as specified, you need to reset in between (and ideally also test number of calls):

```suggestion
    updateStory.mockClear();
    
    fireEvent.change(slider, {
      target: { value: 1 },
    });

    await waitFor(() => {
      expect(updateStory).toHaveBeenCalledWith({
        properties: {
          defaultPageDuration: 1,
        },
      });
      expect(updateStory).toHaveBeenCalledTimes(1);
    });
```